pom.xml

```diff
@@ -1091,6 +1109,12 @@
         <artifactId>hadoop-client-runtime</artifactId>
         <version>${hadoop.version}</version>
         <scope>${hadoop.deps.scope}</scope>
+        <exclusions>
+          <exclusion>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+          </exclusion>
+        </exclusions>
```

Comment:
Is it really unused in `hadoop-client-runtime`?

cc @sunchao

Reply:
This is a runtime dependency for `hadoop-client-runtime`, so I'm not sure if we can remove it.

Reply:
> This is a runtime dependency for `hadoop-client-runtime`, so I'm not sure if we can remove it.

I can test tis since we also have a env of hadoop-3.2.1 and hadoop-3.3.0. I will test this later

Reply:
> Is it really unused in `hadoop-client-runtime`?
> 
> cc @sunchao

Seems ok.

Reply:
> This is a runtime dependency for `hadoop-client-runtime`, so I'm not sure if we can remove it.

Test use `buildin` and `maven` of hadoop-3.2 and hive-2.3.7,  create  dir and write data is ok.

Reply:
I'm always worried at some point something adds the slf4j -> commons-logging binding and then we have something circular here :) this might be fine. Is it helpful, like does it help get more log messages to the same place?